ResNet_constructor.ipynb

```diff
@@ -0,0 +1,1288 @@
+{
```

Comment:
<p>Why do you need those prints? It's huge and not very informative, especially since you now have the dataframe with the same values</p><p>I'd suggest to remove them and write one line at a time, using <code>IPython.display.clear_output</code>  to clear output after each iteration.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bestetc/May/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>